includes/init.php

```diff
@@ -25,16 +35,30 @@ function tsml_archive_template($template)
     add_filter('single_template', 'tsml_single_template');
     function tsml_single_template($template)
     {
-        global $post;
+        global $post, $tsml_user_interface;
         if ($post->post_type == 'tsml_meeting') {
             $user_theme_file = get_stylesheet_directory() . '/single-meetings.php';
-            if (file_exists($user_theme_file)) {
-                return $user_theme_file;
+             
+            if ($tsml_user_interface == 'tsml_ui') { 
+                    // output we are looking for: https://gutenberg.dev.cc/meetings?meeting=never-stand-alone-group  or  https://gutenberg.dev.cc/meetings?meeting=807-1 
+                $mtg_permalink = get_post_type_archive_link('tsml_meeting');
+                wp_redirect(add_query_arg('meeting', $mtg_permalink ));
             }
+            
 
             return dirname(__FILE__) . '/../templates/single-meetings.php';
         } elseif ($post->post_type == 'tsml_location') {
             $user_theme_file = get_stylesheet_directory() . '/single-locations.php';
+            
+            if ($tsml_user_interface == 'tsml_ui') { 
+                $mtg_permalink = get_post_type_archive_link('tsml_meeting');
+                wp_redirect(add_query_arg('meeting', $mtg_permalink ));
+            }
+            
+            if (file_exists($user_theme_file)) {
+                return $user_theme_file;
+            }
```

Comment:
there are two of these one after the next. i believe one of these belongs in the if statement above. otherwise we'd be removing the ability to override the meetings detail page